src/Components/CreateImageWizardV2/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -57,7 +68,29 @@ export const ImageOutputList = () => {
   const arch = useAppSelector((state) => selectArchitecture(state));
   return (
     <TextContent>
-      <ReleaseLifecycle />
+      {distribution === RHEL_8 && (
+        <>
+          <Text className="pf-v5-u-font-size-sm">
```

Comment:
I think we can apply DRY (Don’t Repeat Yourself) a bit to this PR so it is easier to maintain in the future. This code is shared with the `<ReleaseLifecycle>` component. Can you try to extract the common logic to a new component that this component and `<ReleaseLifecycle>` can share.

Reply:
the thing is that this part of the code  {RELEASES.get(release)} will be supported through{' '}
            {toMonthAndYear(RHEL_8_FULL_SUPPORT[0])}, with optional ELS support
            through {toMonthAndYear(RHEL_8_MAINTENANCE_SUPPORT[0])}. Consider
            building an image with {RELEASES.get(RHEL_9)} to extend the support
            period. exist in the old wizard only in case the  (currentStep.name === 'review') 
            how we can handle that with new component?

Reply:
so for that piece of code I think this is the correct place. becuase its apeear only once in case we are at the review step

Reply:
Oh I see, you're right. I noticed it was being used by the same component that is on the Image Output step last night but didn't notice there was a conditional causing it to only be displayed on the Review step. Sorry for the false alarm. :sweat_smile: